superset/models/slice.py

```diff
@@ -160,11 +160,13 @@ def datasource_link(self) -> Optional[Markup]:
 
     @renders("datasource_url")
     def datasource_url(self) -> Optional[str]:
-        # pylint: disable=no-member
         if self.table:
             return self.table.explore_url
         datasource = self.datasource
-        return datasource.explore_url if datasource else None
+        try:
```

Comment:
can we move this logic to the model instead in the Slice?

This issue is happening since the Query object doesn't have a definition for `explore_url` so we need to add the implementation to the `ExploreMixin`